### PR TITLE
qa-fire that might work

### PR DESCRIPTION
**Pull request by tobyhede**

### CODE REVIEW

**Comment by tobyhede** [comment]
Conflict in package.json

**Comment by tobyhede** [comment]
Hmmm ... how to fork a fork and merge back ...

File: webpack.config.babel.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import autoprefixer from 'autoprefixer';
 
 import * as CONFIG from './client/config/_config';
 const projectName = require('./package').name;
-const DEBUG = !process.argv.includes('--release');
+const DEBUG = !process.argv.indexOf('--release') !== -1;
```
Comment (elisechant) [inline webpack.config.babel.js]: this needs rewrite.

File: package.json [hunk 2]
```diff
@@ -143,7 +143,7 @@
     ]
   },
   "engines": {
-    "node": "6.3.0"
+    "node": "6.7.0"
```
Comment (elisechant) [inline package.json]: seems important to flag. why are we changing the node version?

File: webpack.config.babel.js (hunk 1 above)
Comment (elisechant) [inline webpack.config.babel.js]: @maxious what are you trying to achieve here?

File: package.json (hunk 2 above)
Comment (tobyhede) [inline package.json]: @elisechant The buildpack no longer supports 6.3

File: package.json [hunk 3]
```diff
@@ -2,6 +2,7 @@
   "name": "dto-dashboard",
   "version": "1.0.0-beta",
   "description": "DTO Dashboard (beta)",
+  "preferGlobal": true,
```
Comment (tobyhede) [inline package.json]: We're building assets on CF, which means the CF container needs access to all the installed libs

**Pull request merged**